components/googleMap/googleMap.js

```diff
@@ -0,0 +1,306 @@
+import React, { Component } from 'react';
+import { PropTypes } from 'prop-types';
```

Comment:
@fahad19 yep, I fixed it. Let's keep using as a default export :)
but interesting to know :)